source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputLabelView.mm

```diff
@@ -87,6 +87,7 @@ - (instancetype)initInputLabelView:(ACRView *)rootView acoConfig:(ACOHostConfig
         }
 
         self.label.attributedText = attributedLabel;
+        self.labelText = attributedLabel;
 
         std::string errorMessage = inputBlck->GetErrorMessage();
         if (!errorMessage.empty()) {
```

Comment:
@jwoo-msft is this the case where we need to append the error message to the label as well? Are we making hte change here to tackle #4912 also?

Reply:
@shalinijoshi19, I had created a separate branch off this branch for #4912. I just merged it into this branch for CR purposes.